openwisp_radius/models.py

```diff
@@ -97,3 +102,23 @@ class PhoneToken(AbstractPhoneToken):
     class Meta(AbstractPhoneToken.Meta):
         abstract = False
         swappable = swappable_setting('openwisp_radius', 'PhoneToken')
+
+
+class RegisteredUser(AbstractRegisteredUser):
+    class Meta(AbstractRegisteredUser.Meta):
+        abstract = False
+        swappable = swappable_setting('openwisp_radius', 'RegisteredUser')
+
+
+def auto_create_registered_user(sender, instance, created, **kwargs):
+    if created:
+        RegisteredUser.objects.create(user=instance)
+
+
+# Create a new RegisteredUser object for every user
+signals.post_save.connect(
+    auto_create_registered_user,
+    sender=User,
+    weak=False,
+    dispatch_uid='models.auto_create_registered_user',
+)
```

Comment:
I don't think we should do this, we want `RegisteredUser` objects be created only for users who register through the registration API.